var/spack/repos/builtin/packages/py-versioneer/package.py

```diff
@@ -0,0 +1,26 @@
+# Copyright 2013-2020 Lawrence Livermore National Security, LLC and other
+# Spack Project Developers. See the top-level COPYRIGHT file for details.
+#
+# SPDX-License-Identifier: (Apache-2.0 OR MIT)
+
+from spack import *
+
+
+class PyVersioneer(PythonPackage):
+    """Versioneer is a tool to automatically update version strings (in
+    setup.py and the conventional ‘from PROJECT import _version’ pattern) by
```

Comment:
```suggestion
    setup.py and the conventional 'from PROJECT import _version' pattern) by
```
Python 2 can't handle unicode characters in source code, so this will probably cause the Python 2 tests to break.